src/app/exams/exams-view.component.ts

```diff
@@ -27,6 +27,7 @@ export class ExamsViewComponent implements OnInit, OnDestroy {
   maxQuestions = 0;
   answer = new FormControl(null, this.answerValidator);
   incorrectAnswer = false;
+  examPreviewStatus = '';
```

Comment:
Let's rename this variable to `statusMessage` and the two possible values for it should be `incorrect` and `complete`